aspnetcore/index.md

```diff
@@ -71,6 +71,22 @@ Many of the articles and tutorials include links to sample code.
 1. Unzip the *Docs-master.zip* file.
 1. Use the URL in the sample link to help you navigate to the sample directory.
 
+To demonstrate multiple scenarios, sample apps make use of the [&num;define] and [&num;if-&num;else/&num;elif-&num;end-if](/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if) C&num; statements to selectively compile and run different sections of sample code. For those samples that make use of this approach, set the `#define` statement at the top of the C&num; files to the symbol associated with the scenario that you want to run. A sample may require you to set the symbol at the top of multiple files in order to run a scenario.
```

Comment:
Looks like [define] was going to be a link but you left off the URL in parens.  Better than making it a link, though make it code style and make the following link code style too instead of a link; you have "for more info" links already at the end of the section

Reply:
Also, IMHO escaping special characters is more readable than encoding them:  -- e.g., `\#if` instead of `&num;if`

Reply:
I agree  `\#if` much more readable for customers, me, everyone than `&num;if`

@scottaddie 	can we form a consensus on that?

Reply:
I'm old 👴, so it's not surprising that I've been a bit resistant ... a little bit ... to the whole we need a whole new language to convert into the language that we use to make webpages.

I'll put in a patch PR to fix this up.